src/app/SharpRaven/Data/SentryUserFactory.cs

```diff
@@ -85,7 +86,8 @@ private static dynamic GetIpAddress()
         {
             try
             {
-                return SentryRequestFactory.HttpContext.Request.UserHostAddress;
+                return SentryRequestFactory.HttpContext.Request.Headers.GetValues("X-Forwarded-For")?.FirstOrDefault()
```

Comment:
It shouldn't be like that. The client could just send this header and he will hide his real IP.
It should be with some boolean property like `useXForwardedForHeaderIfExist` **with default value `false`**

Reply:
Well, I thought that is not a critical issue for Sentry events from a security standpoint.
You can always get the `real IP` in Headers.
The `useXForwardedForHeaderIfExist` option also is not a perfect solution.
The problem can be solved with a whitelist of reverse proxy IPs, but as I said before, this is not a critical issue.